libs/dh/shared/assets/src/assets/login-page/login-unified.html

```diff
@@ -60,7 +60,7 @@
     <style>
```

Comment:
Please revert this, as this static page doesn't include watt. This is at Mighty ducs backlog, so will be refactored later on. Same with `libs/dh/shared/assets/src/assets/signup-page/signup-unified.html`

Reply:
Reverted the files :)